modules/ROOT/pages/transform-change-target-output-transformation-studio-task.adoc

```diff
@@ -1,21 +1,24 @@
-= To Change the Target Output of a Transformation (Anypoint Studio)
+= Change the Target Output of a Transformation (Anypoint Studio)
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
 By default, the Transform Message component outputs to the message payload. You can change this target, so that the result of your transformation populates another part of the output such as a property or variable.
 
-. Click the *Edit Current Target* button image:using-dataweave-in-studio-47984.png[] above the DataWeave code editor.
+. Click *Edit Current Target* above the DataWeave code editor.
++
+image:using-dataweave-in-studio-47984.png[]
++
 . Specify where in the output Mule message to place the result of your DataWeave transform. In case you're creating a new variable or property, you must also set a name for it.
 +
 image::dw-new-variable.png[new variable]
 
 == See Also
 
-xref:transform-component-about.adoc[About the Transform Component]
+xref:transform-component-about.adoc[Transform Message Component]
 
-xref:transform-add-another-output-transform-studio-task.adoc[To Add Another Output to the Transform Component]
+xref:transform-add-another-output-transform-studio-task.adoc[Add Another Output to the Transform Message Component]
 
 xref:mule-message-structure.adoc[Mule Message Structure]
 
-xref:dataweave.adoc[About DataWeave]
\ No newline at end of file
+xref:dataweave.adoc[DataWeave Language]
```

Comment:
DataWeave script?

Reply:
"DataWeave Language" is correct there. The title of dataweave.adoc is "DataWeave Language".

Reply:
Compare the title in dataweave.adoc:
https://github.com/mulesoft/docs-mule-runtime/edit/DOCS-2552-components-duke/modules/ROOT/pages/dataweave.adoc?pr=%2Fmulesoft%2Fdocs-mule-runtime%2Fpull%2F5
to the title in dataweave-language-introduction.adoc:
https://github.com/mulesoft/docs-mule-runtime/edit/DOCS-2552-components-duke/modules/ROOT/pages/dataweave-language-introduction.adoc?pr=%2Fmulesoft%2Fdocs-mule-runtime%2Fpull%2F5